python/pysrc/kaskada/_timestream.py

```diff
@@ -269,6 +300,28 @@ def __rsub__(self, lhs: Union[Timestream, Literal]) -> Timestream:
             lhs = Timestream._literal(lhs, self._ffi_expr.session())
         return lhs.sub(self)
 
+    def exp(self) -> Timestream:
+        """
+        Create a Timestreamp raising e to the power of this.
+
+        Returns
+        -------
+        Timestream
+            The Timestream resulting from `e^power`.
+        """
+        return Timestream._call("exp", self)
+
+    def floor(self) -> Timestream:
+        """
+        Create a Timestream for the number rounded down to the previous largest integer.
```

Comment:
I wonder if we should tweak the wording. "Previous largest integer" suggests that if the current value *is* an integer, then the result is the previous largest integer (eg., `n - 1`). We maybe just want something like one of:

1. "Create a Timestream of the values rounded down to the nearest integer."
2. "Create a Timestream of the largest integers less than or equal to each point."

(similar for `ceil`)